codespell_lib/tests/test_dictionary.py

```diff
@@ -73,8 +75,8 @@ def _check_aspell(phrase, msg, in_aspell, fname):
         for word in phrase.split():
             _check_aspell(word, msg, in_aspell, fname)
         return  # stop normal checking as we've done each word above
-    this_in_aspell = speller.check(
-        phrase.encode(speller.ConfigKeys()['encoding'][1]))
+    this_in_aspell = any(sp.check(phrase.encode(
+        sp.ConfigKeys()['encoding'][1])) for sp in speller_array)
```

Comment:
The array of languages should vary by dictionary and indeed part of dictionary (to match the in_aspell state). So `dictionary_en-GB_to_en-US.txt` should have the error in the GB dictionaries and the correction(s) in the US dictionaries. Whereas most of the other dictionaries should probably check all variants for errors that aren't and replacements that should be.

Reply:
you can now set in_aspell to None, True, False and a list of dictionaries, that the word should be in.